packages/drivers/routerlicious-driver/src/restWrapper.ts

```diff
@@ -131,6 +132,14 @@ export class RouterliciousRestWrapper extends RestWrapper {
 		statusCode: number,
 		canRetry = true,
 	): Promise<IR11sResponse<T>> {
+		// Check whether this request has been made before or if it is a retry.
+		const originalRequestUrl = requestConfig.url;
```

Comment:
might need to take into account `requestConfig.baseURL` and `requestConfig.params` here